src/rdb/rdb_internal.h

```diff
@@ -86,7 +89,8 @@ struct rdb {
 	int			d_nevents;	/* d_events queue len from 0 */
 	ABT_cond		d_events_cv;	/* for d_events enqueues */
 	uint64_t		d_compact_thres;/* of compactable entries */
-	ABT_cond		d_compact_cv;	/* for base updates */
+	ABT_cond		d_compact_cv;	/* for requesting base updates */
+	ABT_cond		d_compacted_cv;	/* for confirming base updates */
```

Comment:
[Not requesting changes] Strictly speaking, rdb_raft_trigger_compaction changes the log base, that is, there are no "request" and "confirm" steps. So `d_compact_cv` is for base changes, while `d_compacted_cv` is for `dlr_aggregated` changes. Is that right?

Reply:
Thank you.